### PR TITLE
Add mutual references among the mp blogs

### PR DESCRIPTION
**Pull request by Emily-Jiang**

### CODE REVIEW

File: posts/2021-03-19-microprofile40-open-liberty-21003.adoc [hunk 1]
```diff
@@ -107,7 +107,9 @@ Quite a few nice capabilities were provided by MicroProfile Config 2.0, such as
 
 Open Liberty's MicroProfile Config 2.0 feature is based on link:https://github.com/smallrye/smallrye-config[SmallRye's implementation] of the specification.
 
-Watch this space for a new blog post dedicated to MicroProfile Config 2.0.
+Refer to this following dedicated blog on MicroProfile Config 2.0 on how to use the newly added awesome features.
```
Comment (dmuelle) [inline posts/2021-03-19-microprofile40-open-liberty-21003.adoc:110]: suggested edit:

For more information about how to use the awesome new features, check out the link:{url-prefix}/blog/2021/03/31/microprofile-config-2.0.html[MicroProfile Config 2.0 blog post].

File: posts/2021-03-19-microprofile40-open-liberty-21003.adoc [hunk 2]
```diff
@@ -442,7 +444,9 @@ MicroProfile JWT 1.2 adds a number of MicroProfile Config properties to control
 
 MicroProfile JWT 1.2 also supports the JWT specified in a cookie.
 
-Watch this space for a more detailed blog on MicroProfile JWT 1.2.
+Check out the following blog on MicroProfile JWT 1.2 to learn all of the exciting features.
```
Comment (dmuelle) [inline posts/2021-03-19-microprofile40-open-liberty-21003.adoc:447]: For more information about the exciting new features, check out the link:{url-prefix}/blog/2021/03/26/MP-JWT-1.2.html[MicroProfile JWT 1.2 blog post].

Note the updated URL- the current one points to Rest Client

**Review by dmuelle — CHANGES_REQUESTED** [review]
a couple recommendations for blog references, otherwise looks good

File: posts/2021-03-19-microprofile40-open-liberty-21003.adoc (hunk 1 above)
Comment (jakub-pomykala) [reply posts/2021-03-19-microprofile40-open-liberty-21003.adoc:110]: Done

File: posts/2021-03-19-microprofile40-open-liberty-21003.adoc (hunk 2 above)
Comment (jakub-pomykala) [reply posts/2021-03-19-microprofile40-open-liberty-21003.adoc:447]: Done

**Pull request closed without merging**